packages/katello/katello/backup.rb

```diff
@@ -123,7 +123,9 @@ def configure_configs
       feature_configs += ["/var/lib/dhcpd", "/etc/dhcp"] if feature_included?("dhcp")
       feature_configs.push("/usr/share/xml/scap") if feature_included?("openscap")
 
-      backup_configs + feature_configs
+      all_configs = backup_configs + feature_configs
+      all_configs = all_configs.compact
+      all_configs.select { |path| Dir.glob(path).any? }
```

Comment:
nit: I'd merge this line with the previous one

Reply:
fair, updated